js/moveset_import.js

```diff
@@ -20,6 +20,10 @@ function ExportPokemon(pokeInfo) {
 	if (gen > 2) {
 		finalText += "EVs: ";
 		var EVs_Array = [];
+		if (pokemon.HPEVs != -1 && pokemon.HPEVs > 0) { // Do HP EVs exist and are they greater than 0?
```

Comment:
? Does ``if (pokemon.HPEVs && pokemon.HPEVs > 0)`` not work?

Reply:
It does, I just forgot it could be written that way...